buildSrc/src/main/groovy/net.consensys.zkevm.kotlin-common-minimal-conventions.gradle

```diff
@@ -24,6 +24,7 @@ repositories {
     url "https://artifacts.consensys.net/public/linea-besu/maven/"
     content {
       includeGroupAndSubgroups('io.consensys')
```

Comment:
unless `io.consensys` is used by other artifacts hosted on this repo, this line can now be removed

Reply:
@fab-10 thanks, will be addressed in the send-bundle [PR](https://github.com/Consensys/linea-monorepo/pull/771)